org.mintguide.WebAppReader.yaml

```
@@ -0,0 +1,62 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/bin/webappreader.sh
```

```suggestion
command: webappreader.sh
```

as previously said this should NOT be a full path.